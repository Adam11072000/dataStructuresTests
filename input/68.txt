init 221
addImage 176
mergeSuperPixels 176 164 175
addImage 190
getHighestScoredLabel 190 12
quit