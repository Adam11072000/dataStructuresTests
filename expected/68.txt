init done.
addImage: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
quit done.
